### PR TITLE
box-settings-refactor

### PR DESCRIPTION
**Pull request by kghamilton89**
* Refactor of **Box Chart** to be in line with new standards.
* Resized images.
* Re-do examples.
* Restructure document.
* Rewrite settings descriptions.
* Several examples were repeated from Settings tables or otherwise self explanatory (how to use two series is demonstrated by Overview chart for example)

### CODE REVIEW

File: widgets/box-chart/README.md [hunk 1]
```diff
@@ -1,61 +1,61 @@
...
+### Widget Settings
 
-![](./images/box-chart-4.png)
+* The settings apply to the `[widget]` section.
+* [Shared](../shared/README.md) `[widget]` settings are inherited.
 
-[![](./images/button.png)](https://apps.axibase.com/chartlab/14cf1974/2/)
+Name |Example |Description |&nbsp;
+:--|:--|:--|:--
+[`percentiles`](#percentiles) | `percentiles = 10%/90%` | Percentile range visualized box portion of the chart.<br>Possible values: `0%` to `100%`.<br>Default value: `25%/75%`.| [↗](https://apps.axibase.com/chartlab/5275c9a3)
```
Comment (rodionos) [inline widgets/box-chart/README.md]: Should it be listed in the Series section, with example refactored similar to percentile-markers, by setting the value for each series separately. What about class, can it be applied for each series separately?

File: widgets/box-chart/README.md [hunk 2]
```diff
@@ -1,61 +1,61 @@
...
+[`class`](#class) | `class = metro` | Alternative visualization style.| [↗](https://apps.axibase.com/chartlab/c147cb06)
```
Comment (rodionos) [inline widgets/box-chart/README.md]: Rule 8) not applied

File: widgets/box-chart/README.md [hunk 3]
```diff
@@ -1,61 +1,61 @@
...
+### Series Settings
 
-![](./images/box-chart-1.png)
+* The settings apply to the `[series]` section.
+* [Shared](../shared/README.md) `[setting]` settings are inherited.
 
-[![](./images/button.png)](https://apps.axibase.com/chartlab/440aafca)
+Name |Example |Description |&nbsp;
+:--|:--|:--|:--
+[`percentile-markers`](#percentile-markers)| `percentile-markers = all`<br>`percentile-markers = top` | Percentiles marker style.<br>Possible values: `all`, `none`, `top`, `bottom`.<br>Default value: `none`.|[↗](https://apps.axibase.com/chartlab/866a4521)
```
Comment (rodionos) [inline widgets/box-chart/README.md]: Copy descriptions from test.md

File: widgets/box-chart/README.md [hunk 4]
```diff
@@ -1,61 +1,61 @@
...
-### Single Series
+### Alert
 
-![](./images/box-chart-2.png)
+![](./images/alert-expression-1.png)
 
-[![](./images/button.png)](https://apps.axibase.com/chartlab/440aafca/7/)
+[![](../../images/button.png)](https://apps.axibase.com/chartlab/84397e8d)
 
 ### `metro` Mode
```
Comment (rodionos) [inline widgets/box-chart/README.md]: better metric class, not mode

Comment (kghamilton89) [reply widgets/box-chart/README.md]: changed

File: widgets/box-chart/README.md (hunk 1 above)
Comment (kghamilton89) [reply widgets/box-chart/README.md]: * Moved percentiles to Series.
* Class setting cannot be applied to individual series, all or none.

**Pull request merged**